build/mvn

```diff
@@ -69,14 +69,23 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  MVN_VERSION="3.3.9"
```

Comment:
This really should be grep'd from the pom file, as is done with scala_version in install_scala.  Also, make sure that the correct maven in downloaded and installed in the case that there isn't any mvn available when this script is run.